web/pdf_page_view_base.js

```diff
@@ -0,0 +1,224 @@
+/* Copyright 2012 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RenderingCancelledException } from "pdfjs-lib";
+import { RenderingStates } from "./ui_utils.js";
+
+export class PDFPageViewBase {
```

Comment:
- Looking at our existing base-classes they all use a slightly different naming convention, so can we please be consistent here?
   ```suggestion
   class BasePDFPageView {
   ```
 - Can we please move the `export` to end of the file for consistency, since we don't really use "inline" exports in the code-base?

Reply:
It it ok if I rename the class without renaming the file, so that the three "page view" files remain together when sorted alfabetically? (it just makes it easier to find them at glance)

Reply:
To me, it does seem slightly confusing if the class- and file-names differ from each other.